src/commands/database-remove.js

```diff
@@ -8,14 +8,17 @@ var api = require("../api");
 var responseToError = require("../responseToError");
 var FirebaseError = require("../error");
 
-var utils = require("../utils");
-var prompt = require("../prompt");
+var utils = require("../lib/utils");
+var querystring = require("querystring");
+var prompt = require("../lib/prompt");
 var clc = require("cli-color");
 var _ = require("lodash");
 
 module.exports = new Command("database:remove <path>")
   .description("remove data from your Firebase at the specified path")
   .option("-y, --confirm", "pass this option to bypass confirmation prompt")
+  .option("-v, --verbose", "show delete progress (helpful for large delete)")
```

Comment:
Flag changes need API review. Please ask Ryan or Tony who can point you to the correct process.

Reply:
Cool, I probably should have done that first.
I will work on that then.